search_results_list.html

```diff
@@ -0,0 +1,34 @@
+<main class="main" role="main">
+  <div class="wrap clearfix">
+    <div class="row recipe-list">
+      <section class="content">
+        <div class="entries row">
+          <!-- Recipe grid square -->
+          {{#each posts}}
+            <div class="entry one-third" style="margin-bottom: 15px">
+              <!-- Recipe picture -->
+              <figure style="height: 169px; overflow: hidden;">
+                <img src="{{ ../baseurl }}/img/recipes/{{ this.title_image }}" alt="{{ this.title }}" />
```

Comment:
Ah, this file is not actually jekyll, it's handlebars. I'd like to find a way to make it the same, but for now the syntax is slightly different.